docs/cli.md

```diff
@@ -0,0 +1,217 @@
+  -c CONCURRENCY, --concurrency CONCURRENCY
+                        Comma-delimited list of concurrency values or ranges <start:end:step> to be used during profiling
+  --perf-analyzer-timeout PERF_ANALYZER_TIMEOUT
+                        Perf analyzer timeout value in seconds.
+  --perf-analyzer-cpu-util PERF_ANALYZER_CPU_UTIL
+                        Maximum CPU utilization value allowed for the perf_analyzer.
+  --perf-analyzer-path PERF_ANALYZER_PATH
+                        The full path to the perf_analyzer binary executable
+  --perf-measurement-window PERF_MEASUREMENT_WINDOW
+                        Time interval in milliseconds between perf_analyzer measurements. perf_analyzer will take measurements over all the requests completed within this time interval.
+  --perf-output PERF_OUTPUT
+                        Enables the output from the perf_analyzer to stdout
+  --triton-launch-mode {local,docker,remote}
+                        The method by which to launch Triton Server. 'local' assumes tritonserver binary is available locally. 'docker' pulls and launches a triton docker container with the specified version. 'remote'
+                        connects to a running server using given http, grpc and metrics endpoints.
+  --triton-docker-image TRITON_DOCKER_IMAGE
+                        Triton Server Docker image tag
+  --triton-http-endpoint TRITON_HTTP_ENDPOINT
+                        Triton Server HTTP endpoint url used by Model Analyzer client. Will be ignored if server-launch-mode is not 'remote'
+  --triton-grpc-endpoint TRITON_GRPC_ENDPOINT
+                        Triton Server HTTP endpoint url used by Model Analyzer client. Will be ignored if server-launch-mode is not 'remote'
+  --triton-metrics-url TRITON_METRICS_URL
+                        Triton Server Metrics endpoint url. Will be ignored if server-launch-mode is not 'remote'
+  --triton-server-path TRITON_SERVER_PATH
+                        The full path to the tritonserver binary executable
+  --triton-output-path TRITON_OUTPUT_PATH
+                        The full path to the file to which Triton server instance will append their log output. If not specified, they are not written.
+  --run-config-search-max-concurrency RUN_CONFIG_SEARCH_MAX_CONCURRENCY
+                        Max concurrency value that run config search should not go beyond that.
+  --run-config-search-max-instance-count RUN_CONFIG_SEARCH_MAX_INSTANCE_COUNT
+                        Max instance count value that run config search should not go beyond that.
+  --run-config-search-disable
+                        Disable run config search.
+  --run-config-search-max-preferred-batch-size RUN_CONFIG_SEARCH_MAX_PREFERRED_BATCH_SIZE
+                        Max preferred batch size value that run config search should not go beyond that.
+```
+Depending on the command line or YAML config options provided, the `profile`
```

Comment:
Can this be organized better, perhaps with use cases and corresponding command line examples .